### DISCUSSION TITLE
Long Live SageTV

### DISCUSSION THREAD

**Discussion by xjim1** [OpenSageTV/Community#136] [General]
Well, I knew this day would happen. I really thought I'd have moved on by now, but for the amount I record programs, I've not been interested in learning a new system. SageTV continues to work for me, and now I find myself looking to install the Sage client. (where's a crazy lookin' emoji when you need one).

Anyway, are kits still available? I realize I'm late to the funeral, and I'm still trying to figure out how to navigate github(for years, now), but I'm just not ready to give you up yet, Sage. Anyone need a crowbar? Those nails are a bitch to pull out of the coffin. :)

**Comment by CptKirk-1701** [comment]
![Screenshot 2024-06-07 073608](https://github.com/OpenSageTV/Community/assets/15445697/c81c1ffa-82dc-4831-ab07-9251d1526ea6)

I've got all the 9.x downloads going back to 9.0.4.  Final/latest is 9.2.6.  I can throw them up on a google drive if you want.

But the actual release are posted here: https://github.com/OpenSageTV/sagetv-windows/releases

**Reply by xjim1** [reply]
That's what I was looking for.  I'll try the latest, but I'll let you know if I need something earlier.  
Thanks!

**Comment by DarkWolf7** [comment]
Man, to this day I haven't found anything that handles playback as well as SageTV. Plex has improved... but it's still not as good. Maybe someday I'll find something that doesn't hang because you want to rewind 10 seconds... but for now, SageTV it is.

**Reply by Zogg44** [reply]
I think this must be very system dependent, because while I really liked SageTV for the features, I had enough playback issues that I had to move on. Most recordings that I downloaded wouldn't play unless I ran them through Handbrake for a quick touchup, which was a manual process. I was using a Win10 HTPC for playback and had all the latest codecs, too, and SageTV server in an Unraid docker. I think with a Windows-based server it would have worked a bit better but for several reasons I didn't want to make that switch (again).

Also, I'm a bit hard of hearing and need captions/subtitles on and that was always somewhat an issue with SageTV depending on the media source. If there was more than one subtitle track then SageTV refused to play any of them for me. The same files played fine in all other media players, and we wound up just using Plex to watch those with, and as you mentioned Plex playback has improved a lot. I have no issues skipping back or forward.

I now use Channels DVR for my recordings and it's simpler, it's supported, and channel skipping is almost perfect. It doesn't require constant care and feeding, no Java issues, the apps work great, and captions don't take up half the screen. TVE is supported natively, and there are other plugins that handle recording from many streaming sources.

I'm not crapping on SageTV, if it meets your needs then it is awesome and the price is right. For me, it just became outdated enough that I needed to move on.
